Source/Core/Matrix4.js

```diff
@@ -1851,13 +1853,13 @@ Matrix4.multiplyByUniformScale = function (matrix, scale, result) {
 };
 
 /**
- * Multiplies an affine transformation matrix (with a bottom row of <code>[0.0, 0.0, 0.0, 1.0]</code>)
+ * Multiplies a proper rigid transformation matrix (with a bottom row of <code>[0.0, 0.0, 0.0, 1.0]</code>)
```

Comment:
This method only scales the top left 3x3 of an affine matrix. This ignores any translation in the affine transformation.